data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
 		"bunks" 245
 		"mass" 940
 		"drag" 16.1
 		"heat dissipation" .4
 		"fuel capacity" 700
-		"cargo space" 530
-		"outfit space" 740
-		"weapon capacity" 300
-		"engine capacity" 180
+		"cargo space" 240
```

Comment:
TBH 240 is a little low imo. I think going to the mule's previous capacity (270) would be a good change, as it would greatly impede my ability to perform other functions (i.e. haul freight) once I am set up as a warship. I currently convert 280 cargo, so I think 270 would reduce the amount of cargo I can convert _below_ 0, which will remove my freighter role and infringe on my warship role. With 240, I think it would affect my ability to be even only a warship too greatly.